prody/proteins/interactions.py

```diff
@@ -160,17 +161,65 @@ def get_energy(pair, source):
     import numpy as np
     import importlib.resources as pkg_resources    
```

Comment:
These imports should be handled with `if PY3K` instead then the rest of the code is the same

```
    if PY3K:
        import importlib.resources as pkg_resources 
    else:
         import pkg_resources
```

I can fix this

Reply:
ok, it's not the same and I'm fixing it now